demos/blockfactory/block_library_view.js

```diff
@@ -59,10 +53,10 @@ var BlockLibraryView = function() {
  */
 BlockLibraryView.prototype.addOption = function(blockType, selected) {
   // Create option.
-  var option = goog.dom.createDom('a', {
-    'id': 'dropdown_' + blockType,
-    'class': 'blockLibOpt'
-  }, blockType);
+  var option = document.createElement('a');
+  option.id ='dropdown_' + blockType;
+  option.classList.add('blockLibOpt');
+  option.innerHTML = blockType;
```

Comment:
we try and avoid innerHtml. What is this? can it be option.textNode = blockType (or whatever it actually is)